sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -2085,24 +2079,22 @@ block_store(AccessorT acc, simd<T, N> vals, simd_mask<1> pred,
 
 /// The semantics, assumptions and restrictions are identical to
 /// (acc-bs-2) defined above.
-template <typename AccessorT, int N, typename Toffset,
-          typename RegionTy = region1d_t<Toffset, N, 1>,
+template <typename AccessorT, int N, typename OffsetObjT, typename RegionTy,
           typename PropertyListT =
               ext::oneapi::experimental::detail::empty_properties_t>
 __ESIMD_API std::enable_if_t<
     detail::is_device_accessor_with_v<AccessorT,
                                       detail::accessor_mode_cap::can_write> &&
     !sycl::detail::acc_properties::is_local_accessor_v<AccessorT> &&
     ext::oneapi::experimental::is_property_list_v<PropertyListT>>
-block_store(AccessorT acc, simd_view<Toffset, RegionTy> vals,
+block_store(AccessorT acc, simd_view<OffsetObjT, RegionTy> vals,
             PropertyListT props = {}) {
-  block_store<Toffset, N>(acc, vals.read(), props);
+  block_store<OffsetObjT, N>(acc, vals.read(), props);
```

Comment:
I think the original code was wrong, the first template parameter is the accessor type and it would never work. I think none of these are getting called. I made an internal tracker to check if that's true.

Reply:
Sorry I mean probably we can remove this entire function because there's no way it was every getting called because the first template argument was always totally wrong